From 23721dded862320384bb714c058c93da628a0c9e Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Thu, 12 Apr 2012 09:45:10 +0800
Subject: ENGR00179408 MX6DL:Increasing CPU voltage for 800MHz/400MHz/200MHz
 work points

It need add 25mV to 800MHz/400MHz/200MHz work points for MX6DL,
otherwise system will crash when cpu freq switch to these work points

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index 80b5e05..ac65b27 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -131,17 +131,17 @@ static struct cpu_op mx6dl_cpu_op_1G[] = {
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
 	 .cpu_podf = 0,
-	 .cpu_voltage = 1100000,},
+	 .cpu_voltage = 1125000,},
 	 {
 	  .pll_rate = 396000000,
 	  .cpu_rate = 396000000,
 	  .cpu_podf = 0,
-	  .cpu_voltage = 1000000,},
+	  .cpu_voltage = 1025000,},
 	{
 	 .pll_rate = 396000000,
 	 .cpu_rate = 198000000,
 	 .cpu_podf = 1,
-	 .cpu_voltage = 1000000,},
+	 .cpu_voltage = 1025000,},
 };
 
 static struct cpu_op mx6dl_cpu_op[] = {
-- 
1.7.9.5

